services/painting-surface-defect-detection-model-service/app/services/inference.py

```
@@ -0,0 +1,208 @@
+import asyncio
```

Ruff (0.12.2)

`asyncio` imported but unused

Remove unused import: `asyncio`

(F401)